### PR TITLE
Teachingkits

### PR DESCRIPTION
**Pull request by alicoding**

### CODE REVIEW

**Comment by alicoding** [comment]
Once I rebased from master this should be smaller...

**Comment by alicoding** [comment]
@alanmoo @gideonthomas I added the admin part so that it'll be easy to add activity and add teaching kit to see the result object.

File: app/learningapi/teaching_kits/serializers.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            'url', 'difficulty', 'duration', 'tags', 'skills',
+            'authors', 'locale',
+        )
+
+
+class TeachingKitSerializer(serializers.ModelSerializer):
+    """
+    Serializes a teaching kit
+    """
+    activities = ActivitySerializer(many=True)
```
Comment (gideonthomas) [inline app/learningapi/teaching_kits/serializers.py]: I think for the simple use case, let's leave this as `StringRelatedField` instead. We can add complex serialization like above when we implement `?expand` queries.

File: app/learningapi/activities/admin.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from django.contrib import admin
+
+from .models import Activity
+
+
+class TeachingKitInline(admin.StackedInline):
+    model = Activity.teaching_kits.through
+    verbose_name = 'TeachingKit'
```
Comment (gideonthomas) [inline app/learningapi/activities/admin.py]: `verbose_name = 'Teaching Kit'`

File: app/learningapi/teaching_kits/views.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+from rest_framework.generics import ListAPIView, RetrieveAPIView
+from rest_framework.pagination import PageNumberPagination
+
+from learningapi.teaching_kits.models import TeachingKit
+from learningapi.teaching_kits.serializers import (
+    TeachingKitSerializer,
+)
+
+
+class TeachingKitsListView(ListAPIView):
```
Comment (gideonthomas) [inline app/learningapi/teaching_kits/views.py]: `TeachingKitListView`

**Comment by alanmoo** [comment]
Looks fine so far, not sure I have the full model visualized though. What's an example of a Teaching Kit on the current site? [This](https://learning.mozilla.org/en-US/activities/web-lit-basics/)?

**Comment by gideonthomas** [comment]
@alanmoo yep that link points to a teaching kit.

**Comment by gideonthomas** [comment]
This looks good to me, I'll leave it to @alanmoo for final review.

**Comment by alanmoo** [comment]
Is there a plan to be able to add activities from a teaching kit view?

**Comment by alicoding** [comment]
@alanmoo yep, we will add inline adding in the admin interface.

**Comment by alanmoo** [comment]
Great, if that's in the plan this gets a 👍 from me

**Pull request merged**